### PR TITLE
my fave keyboard shortcuts blog

### PR DESCRIPTION
**Pull request by hillalex**
:laughing:

### CODE REVIEW

File: content/blog/most-used-keyboard-shortcuts.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ now [IntelliJ](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts.html).) 
+
+Here are my most used keyboard shortcuts, starting with the basics:
+
+### Global
+These shortcuts are absolute essentials and work in most apps (text editors, browsers, email clients, IDEs, etc.)
+ If you don't already use them, spend some time memorising and practising them. It'll save you loads of time in the long run.
+
+* `ctrl-v` - copy
+* `ctr-x` - cut
```
Comment (richfitz) [inline content/blog/most-used-keyboard-shortcuts.md]: `ctr` :arrow_right: `ctrl` (and elsewhere)

File: content/blog/most-used-keyboard-shortcuts.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+with [Resharper](https://www.jetbrains.com/help/resharper/Reference__Keyboard_Shortcuts.html) at the time,
+ now [IntelliJ](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts.html).) 
+
+Here are my most used keyboard shortcuts, starting with the basics:
+
+### Global
+These shortcuts are absolute essentials and work in most apps (text editors, browsers, email clients, IDEs, etc.)
+ If you don't already use them, spend some time memorising and practising them. It'll save you loads of time in the long run.
+
+* `ctrl-v` - copy
```
Comment (richfitz) [inline content/blog/most-used-keyboard-shortcuts.md]: throughout, if you use `<kbd>ctrl</kbd>` you'll get a little keyboardy markup like <kbd>ctrl</kbd>

File: content/blog/most-used-keyboard-shortcuts.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+been a game changer!
+
+<img src="/img/ctrlR.gif" alt="Reverse search in the terminal" />
+    
+### Browsers
+These work in Chrome, Firefox and IE. Again I use these many many times each day.
+
+* `ctrl-tab` - jump between open tabs
+* `ctrl-t` - open a new tab
+* `ctrl-w` - close the current tab
```
Comment (richfitz) [inline content/blog/most-used-keyboard-shortcuts.md]: How about <kbd>ctrl</kbd>-<kbd>shift</kbd>-<kbd>t</kbd> (re-open last closed tab/window)? I use this one all the time and it seems not that well known

**Review by richfitz — COMMENTED** [review]
Love it - how do we set up a taxonomy of tags? I agree it seems like a good thing to do...

File: content/blog/most-used-keyboard-shortcuts.md (hunk 2 above)
Comment (hillalex) [reply content/blog/most-used-keyboard-shortcuts.md]: Yesss thank you! I was wondering how to make them look like that!

File: content/blog/most-used-keyboard-shortcuts.md (hunk 3 above)
Comment (hillalex) [reply content/blog/most-used-keyboard-shortcuts.md]: I did not know that one! Not technically one of my favourites then, by definition, but sounds like a great one to include... Or we should do a series where we each give our favourites :laughing:

**Comment by hillalex** [comment]
> Love it - how do we set up a taxonomy of tags? I agree it seems like a good thing to do...

I'm going to look into adding that and the team page templates today/tmrw

File: content/blog/most-used-keyboard-shortcuts.md (hunk 3 above)
Comment (hillalex) [reply content/blog/most-used-keyboard-shortcuts.md]: In fact I think it'd be fun to do a follow up post where everyone in the team adds 1 or 2 of their most used shortcuts. I think I'll canvas you all and write a follow up post!

**Review by richfitz — APPROVED** [review]

**Pull request merged**